Library/Homebrew/tap.rb

```diff
@@ -1116,6 +1116,7 @@ def audit_exception(list, formula_or_cask, value = nil)
       return false unless list.include? formula_or_cask
       return list[formula_or_cask] if value.blank?
 
+      list[formula_or_cask].include? value if list[formula_or_cask].is_a? Array
```

Comment:
```suggestion
      return list[formula_or_cask].include? value if list[formula_or_cask].is_a? Array
```
I think you need this too, or that comparison is just ignored?

Reply:
Yeah,  the return statement is important for an early return 🫠